DoseMathTests/DoseMathTests.swift

```diff
@@ -105,6 +107,16 @@ class RecommendTempBasalTests: XCTestCase {
         return TimeInterval(hours: 4)
     }
 
+    var automaticDosingIOBLimit: Double {
+        return 2.0 * maxBolus
+    }
+
+    // Tests that were in place before the addition of automaticDosingIOBLimit
+    // should all still succeed so long as
+    //      insulinOnBoard < automaticDosingIOBLimit - (dose returned by test)
+    //  The test with highest dose is a bolus of 2.3 U
+    var insulinOnBoard: Double = 7.69
```

Comment:
This var is being modified locally in tests. It proves a challenge to the reader to know what this is set in a given test as it then depends on the order tests are run. Because actual IOB in real life is going to vary a lot, and we want to exercise a variety of situations, I don't think this should be a file scope variable; each test should just have its own value of IOB that it passes into the methods under test.  For tests that aren't actually testing particular behavior of IOB and IOB limit, passing in 0 is fine.

Reply:
I can certainly move this parameter internal to each of the test.

Reply:
I believe this is resolved with commit [01c775c](https://github.com/LoopKit/Loop/pull/1871/commits/01c775c8c0efa529a7853f9872e48990981289fd)